config/ms.yml

```diff
@@ -4,7 +4,8 @@ idspace: MS
 base_url: /obo/ms
 
 products:
-- ms.owl: https://github.com/HUPO-PSI/psi-ms-CV/releases/latest/download/psi-ms.owl
+- ms.owl: https://github.com/HUPO-PSI/psi-ms-CV/releases/latest/download/ms.owl
+- psi-ms.owl: https://github.com/HUPO-PSI/psi-ms-CV/releases/latest/download/psi-ms.owl
```

Comment:
These products must all be of the form "id_space.file_extension". The ID_SPACEs are governed by OBO, and the relevant one here is "MS"/"ms". You're not allowed to add a new ID_SPACE here with "psi-ms.owl".